html/changelogs/sonicgotnuked-autolathe.yml

```diff
@@ -0,0 +1,7 @@
+author: SonicGotNuked
+
+delete-after: true
+
+changes:
+  - rscadd: "Adds the implanter to protolathe with corsponding material value and research."
+  - rscadd: "Adds debugger to cargo for 100 credits"
```

Comment:
That changelog line is incorrect.
What you did is make central pay you 100 credits for a shipped debugger (/datum/export)